src/app/meshes/views/MeshOverviewView.vue

```diff
@@ -202,6 +203,8 @@ async function loadMesh(): Promise<void> {
   try {
     mesh.value = await kumaApi.getMesh({ name })
     meshInsights.value = await kumaApi.getMeshInsights({ name })
+    const meshInsight = mergeInsightsReducer([meshInsights.value])
+    store.state.currentMeshPolicies = meshInsight.policies
```

Comment:
**Comment**: As far as I can tell, this is the last use of always loaded mesh insights data. It’s used in the NotificationManager and thus always needed as it’s in our top bar. I tried to keep the change here to a minimum to make it clear exactly _what_ part of mesh insights we’re using (i.e. the `policies` field).